exercises/secret-handshake/introduction.md

```diff
@@ -0,0 +1,12 @@
+# Introduction
+
+You are making a secret coding club with some friends and friends-of-friends.
+You don't know everyone in the club, so you and your friends have decided to create a secret handshake that you can use to recognise that someone is a member.
+You don't want anyone who isn't in the know to be able to crack the code.
+
+You designed the code so that it takes a binary number and turns it into a series of actions.
```

Comment:
```suggestion
You designed the code so that one person says a number between 1 and 30, and the other other person turns it into a series of actions.
```

Reply:
```
> 31.to_s(2)
=> "11111"
> 30.to_s(2)
=> "11110"
```

Reply:
Fixed this.